platform/javascript/engine.js

```diff
@@ -273,6 +261,20 @@
 
 	Engine.RuntimeEnvironment = engine.RuntimeEnvironment;
 
+	Engine.isWebGLAvailable = function(majorVersion = 1) {
+
+		var testContext = false;
+		try {
+			var testCanvas = document.createElement('canvas');
+			if (majorVersion === 1) {
+				testContext = testCanvas.getContext('webgl') || testCanvas.getContet('experimental-webgl');
+			} else if (majorVersion === 2) {
+				testContext = testCanvas.getContext('webgl2') || testCanvas.getContet('experimental-webgl2');
```

Comment:
@eska014: Looks like there are 2 missing `x`s in `testCanvas.getContet`

Reply:
Thanks for the tip, fixed